futurenhs.automation/features/foldersManagement.feature

```diff
@@ -1,134 +1,134 @@
-Examples: 
-    | foldertitle   | description    | errorMsg                       |
-    |               |                | Enter the folder title         |
-    | Empty Folder  |                | Enter a unique folder title    |
-    | [STRING: 201] | description    | Enter 200 or fewer characters  |
-    | FolderName    | [STRING: 4001] | Enter 4000 or fewer characters |
-
-@Pending
-# FEATURE NOT YET IN PRIVATE BETA
-Scenario: FNHS70 - Delete folder, Cancel action
-    When I click the 'DeleteFolder' link
-    Then the 'DeleteFolder' header is displayed
-    When I click the 'Delete Folder' link
-    Then the 'Folder will be deleted' header is displayed
-    And the 'Any folder contents will also be discarded. Are you sure you wish to proceed?' textual value is displayed
-    When I cancel this on the open 'Delete Folder' dialog
-    Then the 'DeleteFolder' header is displayed
-
-@Core @Pending
-# FEATURE NOT YET IN PRIVATE BETA
-Scenario: FNHS71 - Delete a folder
-    When I click the 'DeleteFolder' link
-    Then the 'DeleteFolder' header is displayed
-    When I click the 'Delete Folder' link
-    Then the 'Folder will be deleted' header is displayed
-    And the 'Any folder contents will also be discarded. Are you sure you wish to proceed?' textual value is displayed
-    When I confirm this on the open 'Delete Folder' dialog
-    Then the 'Files' header is displayed
-    Then the 'AutoFolderEdited' link is not displayed
\ No newline at end of file
+    User journeys covering functionality and navigation around forums as a Site/Forum Administrator
+
+    Background:
+        Given I have navigated to '/'
+        And I have logged in as a 'group admin' and accept the cookies
+        Then the 'My Groups' header is displayed
+        When I select the 'Automation Admin Group' group card
+        Then the 'Automation Admin Group' header is displayed
+        When I click the 'Files' tab
+        Then the 'Files' header is displayed
+
+
+    Scenario: FNHS64 - Group Files tab page validation
+        Then the 'Add Folder' link is displayed
+        And the 'Group Files' table is displayed
+            | Type | Name                   | Description                        | Modified |
+            |      | Automation Test Folder |                                    |          |
+            |      | DeleteFolder           | Folder to be deleted by automation |          |
+            |      | EditableFolder         | Folder to be edited by automation  |          |
+            |      | Empty Folder           | Empty Folder for testing           |          |
+
+
+    Scenario: FNHS65 - Folder page validation
+        When I click the 'Automation Test Folder' link
+        Then the 'Automation Test Folder' header is displayed
+        Then the 'Upload File' link is displayed
+        And the 'Add Folder' link is displayed
+        And the 'Edit Folder' link is displayed
+        # And the 'Delete Folder' link is displayed
```

Comment:
Discussed on call, that this is currently deprecated until feature goes back in so this is the reason why it is still there and not fully removed.